initfiles/componentfiles/thor/start_backupnode.in

```diff
@@ -71,7 +71,7 @@ if [ ! -z ${THORPRIMARY} ]; then
 else
     groupName=${THORNAME}
 fi
-daliadmin $DALISERVER dfsgroup ${groupName} > $INSTANCE_DIR/slaves
+daliadmin $DALISERVER dfsgroup ${groupName} $INSTANCE_DIR/backupnode.slaves
```

Comment:
Actually that change is already done in another [merged] PR (targeted at 5.4) - HPCC-13272

I will mean that when this is upmerged it will clash